binary_transparency/firmware/devices/usb_armory/README.md

```diff
@@ -40,7 +40,7 @@ index | name       | size    | format | notes
 ------|------------|---------|--------|-----------------------------------------------
 1     | boot       | 10M     | raw    | Must cover disk bytes 1024 onwards as we'll directly write the bootloader here.
 2     | proof      | 512KB   | ext4   | EXT4 filesytem for storing a serialised proof bundle
-3     | firmware   | 64MB+   | ext4   | EXT4 filesystem containing the bootable firmware  image, armory boot config, and DTD file.
+3     | firmware   | 64MB+   | ext4   | EXT4 filesystem containing the bootable firmware image, armory boot config, etc.
```

Comment:
Just a thought, better to specify what is relevant to us rather than etc. This makes the statement concise as done in the earlier text, before mods.

Reply:
The problem is that the DTB file is not necessary for unikernels, it's also perfectly fine to include other files here which the unikernel/Linux kernel are configured to themselves require once booted. 
Given this is a summary table explaining the purpose of the partitions, I think it's ok to give the flavour here, and then the concrete details about what, exactly, is necessary for each type of firmware are present below.